components/EditForm.js

```diff
@@ -108,3 +111,10 @@ const StyledTextarea = styled.textarea`
   padding: 10px;
   border-radius: 10px;
 `;
+
+const StyledImage = styled.div`
+  position: absolute;
```

Comment:
I'm a little unsure what this is referring to. I can't seem to find the corresponding `relative` to this `absolute`.

Reply:
I have added the relative to the 'StyledFormContainer' now. Thank you for noticing :3